src/nplinker/metabolomics/load_gnps.py

```diff
@@ -19,20 +26,41 @@
 GNPS_FORMAT_NEW_FBMN = 'fbmn'
```

Comment:
Wrap GNPS_FORMAT_* as a `Enum` class?

Reply:
Better to provide some background info (or link) about the different formats explain e.g. how `allfiles` format look like